sky/skylet/events.py

```diff
@@ -120,13 +119,15 @@ def _stop_cluster(self, autostop_config):
         if (autostop_config.backend ==
                 cloud_vm_ray_backend.CloudVmRayBackend.NAME):
             self._replace_yaml_for_stopping(self.ray_yaml_path,
-                                            autostop_config.terminate)
+                                            autostop_config.down)
             # `ray up` is required to reset the upscaling speed and min/max
             # workers. Otherwise, `ray down --workers-only` will continuously
             # scale down and up.
-            subprocess.run(
-                ['ray', 'up', '-y', '--restart-only', self.ray_yaml_path],
-                check=True)
+            subprocess.run([
+                'ray', 'up', '-y', '--restart-only', '--disable-usage-stats',
```

Comment:
Nice! 

Should we add this to 
```

sky/backends/cloud_vm_ray_backend.py
1129:                ['ray', 'up', '-y', '--no-restart', cluster_config_file],
1337:            ['ray', 'up', '-y', '--restart-only', handle.cluster_yaml],
```
as well?

Reply:
Good point! Added. For thos two places, I use the environemnt variable instead, as the ray version on the user's laptop can be lower than 1.13, which does not have `--disable-usage-stats` option.